mmaction/datasets/audio_visual_dataset.py

```diff
@@ -65,7 +65,7 @@ def load_annotations(self):
                     idx += 1
                 # idx for label[s]
                 label = [int(x) for x in line_split[idx:]]
-                assert len(label), f'missing label in line: {line}'
+                assert len(label) != 0, f'missing label in line: {line}'
```

Comment:
maybe `> 0` is better, but it's not a big problem

Reply:
just format according to Pylint which is based on PEP8